tsconfig.json

```diff
@@ -1,15 +1,20 @@
 {
-  "compilerOptions": {
-    "target": "es5",
-    "module": "commonjs",
-    "outDir": "./dist",
-    "strict": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
-    "noImplicitReturns": true,
-    "noFallthroughCasesInSwitch": true,
-    "esModuleInterop": true
-  },
-  "include": ["src/**/*"],
-  "exclude": ["node_modules", "**/*.test.ts"]
-}
\ No newline at end of file
+    "compilerOptions": {
+        "target": "ES2019",
```

Comment:
esse target aqui é meio perigoso hein.. vai depender da versão do node da pessoa

Reply:
Nosso projeto só roda com node 12 pra cima por conta da lib do discord, então é compativel.

https://stackoverflow.com/questions/59787574/typescript-tsconfig-settings-for-node-js-12